cibyl/models/ci/printers/colored.py

```diff
@@ -92,6 +93,21 @@ def print_jobs_system(self, system):
 
         return printer.build()
 
+    def print_tenant(self, tenant):
+        result = IndentedTextBuilder()
+
+        result.add(self._palette.blue('Tenant: '), 0)
+        result[-1].append(tenant.name.value)
+
+        for job in tenant.jobs.values():
+            result.add(self.print_job(job), 1)
+
+        if self.mode == PrintMode.COMPLETE:
```

Comment:
This check should use system.is_queried as in line 87, per #177

Reply:
Great, which reminds me that I totally forgot to add a print function for ZuulSystem. Thanks for the tip.

Reply:
I have just uploaded a new commit with that change.